doc/source/whatsnew/v1.2.0.rst

```diff
@@ -222,12 +222,12 @@ Deprecations
 Performance improvements
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-- Performance improvements when creating DataFrame or Series with dtype `str` or :class:`StringDtype` from array with many string elements (:issue:`36304`, :issue:`36317`, :issue:`36325`, :issue:`36432`)
+- Performance improvements when creating DataFrame or Series with dtype ``str`` or :class:`StringDtype` from array with many string elements (:issue:`36304`, :issue:`36317`, :issue:`36325`, :issue:`36432`)
 - Performance improvement in :meth:`GroupBy.agg` with the ``numba`` engine (:issue:`35759`)
 - Performance improvements when creating :meth:`pd.Series.map` from a huge dictionary (:issue:`34717`)
 - Performance improvement in :meth:`GroupBy.transform` with the ``numba`` engine (:issue:`36240`)
 - ``Styler`` uuid method altered to compress data transmission over web whilst maintaining reasonably low table collision probability (:issue:`36345`)
-- Performance improvement in :meth:`pd.to_datetime` with non-`ns` time unit for `float` `dtype` columns (:issue:`20445`)
+- Performance improvement in :meth:`pd.to_datetime` with non-``ns`` time unit for ``float`` ``dtype`` columns (:issue:`20445`)
```

Comment:
```suggestion
- Performance improvement in :meth:`pd.to_datetime` with non-`ns` time unit for ``float`` ``dtype`` columns (:issue:`20445`)
```

Somewhat pedantic but this seems incorrect to treat as a literal

Reply:
Maybe no backticks at all? I think this is going to fail with single

Reply:
yeah this should be double or nothing